src/version.hpp

```diff
@@ -15,6 +15,6 @@
 //*****************************************************************************
 #ifndef SRC_VERSION_HPP_
 #define SRC_VERSION_HPP_
-#define PROJECT_NAME "REPLACE_PROJECT_NAME"
-#define OPENVINO_NAME "REPLACE_OPENVINO_NAME"
+#define PROJECT_NAME "2021.2.1"
+#define OPENVINO_NAME "2021.2"
```

Comment:
I think it should be 2021.3 for both. Please change macro name to more appropriate
PROJECT_NAME -> OVMS_VERSION?
OPENVINO_NAME -> OV_VERSION?